pyaedt/hfss3dlayout.py

```diff
@@ -407,6 +425,39 @@ def create_ports_on_component_by_nets(
                     ports.append(bound)
         return ports
 
+    @pyaedt_function_handler()
+    def create_pec_on_component_by_nets(
+        self,
+        component_name,
+        nets,
+    ):
+        """Create a PEC connection on a component for a list of nets.
+
+        Parameters
+        ----------
+        component_name : str
+            Component name.
+        nets : str, list
+            Nets to include.
+
+
+        Returns
+        -------
+        bool
+
+        References
+        ----------
+
+        >>> oEditor.CreateEdgePort
+        """
+        if isinstance(nets, list):
+            pass
+        else:
+            nets = [nets]
+        net_array = ["NAME:Nets"] + nets
+        self.oeditor.CreatePortsOnComponentsByNet(["NAME:Components", component_name], net_array, "PEC", "0", "0", "0")
+        return True
+
     @pyaedt_function_handler()
     def create_differential_port(self, via_signal, via_reference, port_name, deembed=True):
         """Create a new differential port.
```

Comment:
```suggestion
        """Create a differential port.
```
Create means "new" so it's not necessary.